Sources/muterCore/Configuration/MuterConfiguration.swift

```diff
@@ -3,18 +3,23 @@ import Foundation
 public struct MuterConfiguration: Equatable, Codable {
     let testCommandArguments: [String]
     let testCommandExecutable: String
+    /// File exclusion list.
     let excludeList: [String]
```

Comment:
If you are creating the `excludeCallList`, it would be nice if you rename this one to `excludeFileList`.

Reply:
Just to not cause any confusion.

Reply:
No problem, I just wanted to reduce the number of changes so that the diff is easier to review. I'll do it as a last commit for this PR.

Reply:
Good approach.

Reply:
Done. I didn't change the name in the configuration file, though, so that existing configuration files work properly.